dali/test/python/test_operator_warp.py

```diff
@@ -124,19 +132,24 @@ def define_graph(self):
 
 
 class CVPipeline(Pipeline):
-    def __init__(self, batch_size, output_type, input_type, use_input, num_threads=3, device_id=0, num_gpus=1, inv_map=False):
-        super(CVPipeline, self).__init__(batch_size, num_threads, device_id, seed=7865, exec_async=False, exec_pipelined=False)
+    def __init__(self, batch_size, output_type, input_type, use_input, num_threads=3, device_id=0,
+                 num_gpus=1, inv_map=False):
+        super(CVPipeline, self).__init__(batch_size, num_threads, device_id, seed=7865,
+                                         exec_async=False, exec_pipelined=False)
         self.use_input = use_input
         self.name = "cv"
-        self.input = ops.readers.Caffe(path=caffe_db_folder, shard_id=device_id, num_shards=num_gpus)
+        self.input = ops.readers.Caffe(path=caffe_db_folder, shard_id=device_id,
+                                       num_shards=num_gpus)
         self.decode = ops.decoders.Image(device="cpu", output_type=types.RGB)
         if self.use_input:
-            self.transform_source = ops.ExternalSource(lambda: gen_transforms(self.max_batch_size, 10))
-            self.warp = ops.PythonFunction(function=CVWarp(output_type, input_type, inv_map=inv_map),
-                                           output_layouts="HWC")
+            self.transform_source = ops.ExternalSource(
+                lambda: gen_transforms(self.max_batch_size, 10))
```

Comment:
```suggestion
            self.transform_source = ops.ExternalSource(lambda: gen_transforms(self.max_batch_size, 10))
```
It's below 100 characters...